wiki/lib/markdown.py

```
@@ -730,42 +749,80 @@ def replace_button(match):
     def replace_tabs(match):
-        blocks = match.group(1).strip()
-        if not _contains_only_pre_blocks(blocks):
+        parts = _TAB_MARKER_HTML_RE.split(match.group(1))
+        # parts = [before_first_marker, label1, body1, label2, body2, …]
+        if len(parts) < 3 or parts[0].strip():
             return match.group(0)
-        return f'<div class="code-tabs">\n{blocks}\n</div>'
-
-    return _CODE_TABS_RE.sub(replace_tabs, html)
+        panels = []
+        for label_html, body in zip(parts[1::2], parts[2::2]):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)